### ISSUE TITLE
Float = false does not seem to make a difference

### ISSUE THREAD

**Issue by DannyBen** [justinwalsh/daux.io#246]
Hi,

This looks like a very nice documentation generator, but I am trying to remove the right code column, and have the code stay in the main content area.

I am trying this with the provided example docs folder, and I have edited its `float` attribute to be false.
I expected the result to be [like this](http://felkerdigitalmedia.com/gltn/docs/index.php?Dictionaries/XML_Dictionary) and instead, I still have the right code column. I am viewing this on a wide desktop screen, Chrome on Windows. I can provide a screenshot if needed, but it looks like the screenshot in issue #25

**Comment by harikt** [comment]
@DannyBen you are right. I was also having the same trouble and your solution fixed it.

**Comment by makailol** [comment]
I am having the same issue. Only changing float to false doesn't make difference. But changing both toggle_code and float to false works.

Is it possible to fix this issue so that "float":false works regardless of toggle_code value ?

**Comment by rain01** [comment]
**tl;dr:** fastest fix is keeping "toggle_code" false and commenting out line from [custom.js line 67](https://github.com/justinwalsh/daux.io/blob/master/js/custom.js#L67) because that's what is overwriting everything. Then users can't toggle code blocks anymore but at least it reads the config correctly and you won't get JS errors.

@DannyBen I did little research and found out that JS fails when `toggle_code` is set to false, so it's definitely a bug.

Why changing float value doesn't matter is because when first loading the page, it will remember the default code block state which is floating code block at right side and is set here: 
https://github.com/justinwalsh/daux.io/blob/master/js/custom.js#L60
That won't happen when JS fails because of `toggle_code` false... otherwise JS will overwrite your preference you set in config.

Also I noticed extra features like this: 
https://github.com/justinwalsh/daux.io/blob/master/js/custom.js#L64
Which will change the float depending if you have any code blocks on the page or not.

**Comment by onigoetz** [comment]
Fixed in development

**Comment by michaelharrer** [comment]
This still does not work, does it? I tried both, development and master branch, but i still have the right-column for the code. With a smaller viewport it looks as desired. 

Here is my config.json 
{
    "title": "DAUX.IO",
    "tagline": "The Easiest Way To Document Your Project",
    "author": "Justin Walsh",
    "image": "app.png",
    "ignore": {
            "files": ["Work_In_Progress.md"],
            "folders": ["99_Not_Ready"]
    },
    "live": {
        "inherit_index": true,
        "clean_urls": true
    },
    "html": {
        "theme": "daux-red",
        "toggle_code": false,
        "float": false
    }
}

**Comment by symera** [comment]
Unfortunately not. Same issue here. That "workaround" with `toggle_code` doesn't work either.

My `config.json` looks like this:
```
{
    "title": "DAUX.IO",
	"tagline": "The Easiest Way To Document Your Project",
	"author": "Justin Walsh",
	"image": "app.png",
    "ignore": {
            "files": ["Work_In_Progress.md"],
            "folders": ["99_Not_Ready"]
    },
    "live": {
        "inherit_index": true,
        "clean_urls": true
    },
    "html": {
        "theme": "daux-red",
        "breadcrumbs": true,
        "breadcrumb_separator": "Chevrons",
        "toggle_code": false,
        "date_modified": true,
        "float": false,

        "links": {
            "Documentation": "./Get_Started"
        }
    }
}
```

**Comment by symera** [comment]
I don't know why exactly but the `toggle_code` and float settings seems to have no effect at all. The class `float-view` won't get removed, no matter what I have set in the `config.json`.

Nevertheless I just found a dirty workaround:
Edit [daux.js @ line 89](https://github.com/justinwalsh/daux.io/blob/master/themes/daux/js/daux.js#L89) by replacing
```
    setCodeBlockStyle(codeBlockState);
```
with
```
    //setCodeBlockStyle(codeBlockState);
    setCodeBlockStyle(1);
```

**Comment by onigoetz** [comment]
Hi, I made some tests on the current version, and it works as expected. 

I just discovered a small issue that happens when the code is floating and then "float" is set to false. In that case when reloading the page, it's still floating.

I fixed this and merged it in the default branch.

**Comment by michaelharrer** [comment]
works like a charm. thanks very much!
